ydb/public/lib/ydb_cli/common/sys.cpp

```diff
@@ -4,14 +4,18 @@
 
 namespace NYdb::NConsoleClient {
 
-bool IsSystemObject(const NScheme::TSchemeEntry& entry) {
-    if (entry.Type != NScheme::ESchemeEntryType::Directory) {
+bool IsSystemDir(const NScheme::TSchemeEntry& entry) {
+    if (entry.Type == NScheme::ESchemeEntryType::Directory) {
+        return entry.Name.starts_with("~")
+            || entry.Name.starts_with(".sys")
+            || entry.Name.starts_with(".metadata");
+    } else {
         return false;
     }
+}
```

Comment:
FYI: This name check is outdated, there are more directories that are considered "system" and which content should not be backed up (and can't be restored).
See `ydb/core/tx/schemeshard/schemeshard_system_names.cpp`.

I don't suggest to use reserved names registry now, in this PR, but eventually it should be done somehow. Thought it should be mentioned.